### PR TITLE
records: add load_resolved_schema

### PR DESCRIPTION
**Pull request by jmartinm**
## Description
New load_resolved_schema function that allows loading a given
schema with all $ref resolved.

## Checklist:
- [x] I have all the information that I need (if not, move to `RFC` and look for it).
- [ ] I linked the related issue(s) in the corresponding commit logs.
- [x] I wrote [good commit log messages](https://github.com/torvalds/subsurface-for-dirk/blob/5f15ad5a86ada3c5e574041a5f9d85235322dabb/README#L92-L119).
- [x] My code follows the code style of this project.
- [x] I've added any new docs if API/utils methods were added.
- [ ] I have updated the existing documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by jmartinm** [comment]
@Glignos this is one way of getting the schema resolved.

**Comment by jmartinm** [comment]
This is by the way, replicating the logic present in https://github.com/inveniosoftware/invenio-jsonschemas/blob/master/invenio_jsonschemas/views.py#L60-L65

Ideally `invenio-jsonschemas` would provide that functionality as a utility function (not only in the view)

File: inspirehep/modules/records/json_ref_loader.py [hunk 1]
```diff
@@ -99,6 +100,21 @@ def get_record(self, pid_type, recid):
 """Used in invenio-jsonschemas to resolve relative $ref."""
 
 
+def load_resolved_schema(name):
+    """Load the given schema from wherever it's installed.
+
+    All $ref will be resolved using SCHEMA_LOADER_CLS
+
+    :param schema_name: Name of the schema to load, for example 'authors'.
```
Comment (jacquerie) [inline inspirehep/modules/records/json_ref_loader.py]: Let's use Napoleon!
```
Arguments:
    name(str): Name of the schema to load.

Examples:
    >>> load_resolved_schema('authors')
    ...
```

**Comment by jmartinm** [comment]
@Glignos merged. You can use it now

**Pull request merged**